packages/cms-lib/lib/config.js

```diff
@@ -27,10 +27,13 @@ let environmentVariableConfigLoaded = false;
 
 const getConfig = () => _config;
 
-const getConfigFilename = () => {
+const getRelativeConfigFilename = () => {
   const configPathSegments = _configPath.split('/');
```

Comment:
FWIW, this code assumes that `/` is a separator. I believe we have code elsewhere that determines the separator based on the operating system.